R/org_classify_description_texts.R

```diff
@@ -283,6 +283,40 @@ get_organization_name_990 <- function(ein) {
   return(organization_name)
 }
 
+#' Search for the EIN an organization given a name and U.S. state 
+#'
+#' @param state_abbrev A two-letter U.S. state abbreviation  
+#' @param org_name The name of a non-profit organization
+#' 
+#' @return If successful, the function returns the Employment Identification Number matching the organization name in the given state
+#' @importFrom dplyr filter
+#' @importFrom dplyr select
+#' @importFrom dplyr arrange
+#' @export
+get_organization_ein <- function(state_abbrev,org_name) {
+  state_code <- as.character(state_abbrev)
+  org_name <- as.character(org_name)
+
+  if (nchar(org_name) < 5) { # require exact match for short names
```

Comment:
@milandv Consider moving the comment to line 299 and enter. The same applies to the comment in line 304.